R/run_plus_attributes.R

```diff
@@ -37,6 +37,12 @@
 add_plus_network_attributes <- function(net, override = 5,
                                         cores = NULL, split_temp = NULL,
                                         status = TRUE) {
+  add_sf <- FALSE
+  if(inherits(net, "sf")) {
+    add_sf <- TRUE
+    geom <- dplyr::select(net)
+    net <- drop_geometry(net)
+  }
```

Comment:
This works with sf now by just pulling it out and adding it back at return time.